Linked List/LinkedList_Intersection_of_two.py

```diff
@@ -0,0 +1,17 @@
+# Definition for singly-linked list.
```

Comment:
Can you explain your solution as well? It will be read by others so it will be good if u add proper explanation with comments in all programs